cypress/integration/FileInputFieldWithList/Deduplicates_the_file_list/index.js

```diff
@@ -0,0 +1,41 @@
+import '../common'
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('the list contains the file duplicate.md', () => {
+    cy.contains('duplicate.md').then($label => {
+        expect($label).to.have.lengthOf(1)
+        expect($label).to.have.class('label')
+        expect(
+            $label.closest('[data-test="dhis2-uicore-filelistitem"]')
+        ).to.have.lengthOf(1)
+    })
```

Comment:
I think we should use cypress' dom assertions when expecting some dom scenarios:
```js
cy.contains('duplicate.md')
  .should('have.lengthOf', 1)
  .should('have.class', 'label')
  .closest('[data-test="dhis2-uicore-filelistitem"]')
  .should('have.lengthOf', 1)
```

But this is not a biggie, so wouldn't mind if you don't change it